### DISCUSSION TITLE
rectangles instead of text in some Asian countries

### DISCUSSION THREAD

**Discussion by pavelorehov** [mapsforge/mapsforge#1212] [Q&A] [answered]
I see rectangles instead of text in some Asian countries Japan or Thai.
mapsforge-map-awt 0.6.0 
Issue happens on both Ubuntu and Alpine so not sure if related to missing fonts in OS.
![GeoAsiaRectangles](https://user-images.githubusercontent.com/4618212/117937288-7a6fe680-b30e-11eb-9e8b-79cf152ac5bd.PNG)

**Comment by devemux86** [comment, marked as answer]
Mapsforge 0.6 is extremely old, you should use the [latest version](https://github.com/mapsforge/mapsforge/blob/master/docs/Changelog.md).

Anyway this seems related to Java & fonts in the operating system.

Testing latest Mapsforge on Ubuntu, it seems to work correctly:

![Cruiser](https://user-images.githubusercontent.com/3484020/117939069-62996200-b310-11eb-875b-bf2169d6dd5e.png)
